packages/shared/styles/components/molecules/SfRange.scss

```diff
@@ -4,7 +4,7 @@
   position: relative;
   width: var(--range-width, 100%);
   height: var(--range-height, 7px);
```

Comment:
7px? set it as `--range-height` default value please

Reply:
ok I changed it.